distro/__main__.py

```diff
@@ -0,0 +1,4 @@
+from . import main
```

Comment:
```suggestion
from distro import main
```

Reply:
IMO the relative import is a better choice. It avoids conflicts with any other "distro" module/package in the Python path. I'd prefer to keep this as-is unless there is a strong reason against it.